Core/Inc/iris_system.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
 
 //#define UART_DEBUG
 #define SPI_DEBUG
+#define SPI_DEBUG_UART_OUTPUT

```

Comment:
![image](https://user-images.githubusercontent.com/61334243/179634733-56bd196b-6415-453f-a8c4-231dd2210f07.png)

Reply:
This was meant to be there for debugging only, I need to see what iris is doing when interacting with obc